cmd-graph.c

```diff
@@ -71,6 +72,103 @@ static bool full_graph = false;
 static struct rb_root tasks = RB_ROOT;
 static struct uftrace_graph *graph_list = NULL;
 
+static void print_total_time(struct field_data *fd)
+{
+	struct graph_node *node = (struct graph_node*)fd->arg;
+	uint64_t d = 0;
+
+	d = node->time;
+
+	print_time_unit(d);
+}
+
+static void print_self_time(struct field_data *fd)
+{
+	struct graph_node *node = (struct graph_node*)fd->arg;
+	uint64_t d = 0;
+
+	d = node->time - node->child_time;
+
+	print_time_unit(d);
+}
+
+static void print_addr(struct field_data *fd)
+{
+	struct graph_node *node = (struct graph_node*)fd->arg;
+
+	/* uftrace records (truncated) 48-bit addresses */
+	int width = sizeof(long) == 4 ? 8 : 12;
+
+	pr_out("%*lx", width, node->addr);
+}
+
+static struct display_field graph_field_total_time= {
+	.id      = GRAPH_F_TOTAL_TIME,
+	.name    = "time",
```

Comment:
I thought the field name "time" can be confused graph total time and replay timestamp
https://github.com/namhyung/uftrace/blob/master/cmd-replay.c#L139-L140

What do you think about it ?

Reply:
I thought about changing it `totaltime` but it's too verbose. And the field name is also too long to put in. I think using `time` and `selftime` is enough.

Reply:
And I prefer having `-` in between.

Reply:
What about each field gets separated by 2 spaces like `uftrace report`?  The time fields already uses a space between time and unit so single space looks not enough IMO.